client/components/EventItem/index.jsx

```diff
@@ -24,14 +22,12 @@ export const EventItem = ({
         session,
     }) => {
     const hasBeenCanceled = get(event, 'occur_status.qcode') === 'eocstat:eos6'
-    const itemLockedInThisSession = isItemLockedInThisSession(event, session)
-
-    const callBacks = { [GENERIC_ITEM_ACTIONS.SPIKE.label]: () => onSpikeEvent(event) }
+    const callBacks = {
+        [GENERIC_ITEM_ACTIONS.SPIKE.label]: () => onSpikeEvent(event),
```

Comment:
Same as with my previous comment, would it be better to bind instead of calling from within an arrow function?